microcosm_flask/swagger/definitions.py

```diff
@@ -135,6 +135,19 @@ def body_param(schema):
     })
 
 
+def header_param(name, required=False, param_type="string"):
```

Comment:
I tested out setting `required` to True for this header (because almost every API will return some null data so it's almost always correct to set this with a Swagger request). Unfortunately, I couldn't get bravado to validate when this happened; it's happy to send the header, but it doesn't connect the header it sends to the Swagger schema.